Drop the dependency info boxes

ennru opened the pull request:
The dependency info for different build tools is nowadays well present in all Akka docs and should be dropped from the overview page

(I'm referring to this kind of box on https://akka.io/docs/)
![bild](https://user-images.githubusercontent.com/458526/53819231-d0e07e00-3f69-11e9-8759-1067bdf87e8f.png)

ennru commented:
One difference to the ones we've on most pages now is that these boxes even listed the relevant testkit dependency.

ennru commented:
@raboof I've left this hanging. Anything we can do to get this in? The page becomes much more beautiful <3

raboof commented:
OK, then let's make sure the dependency is visible on the first page when clicking 'reference' here.

This seems OK for all projects except Alpakka Kafka, Akka HTTP and Akka gRPC right now. Perhaps add them there and then merge this?

patriknw commented:
Sounds good

ennru commented:
I changed the links for Alpakka Kafka and Akka HTTP to point to pages containing the dependency info. For gRPC there is no such page.

A new thing I changed is to just have "Scala and Java" for all modules as the split was from the old docs layout, this spares a redirect.

patriknw commented on docs/index.html:
```diff
@@ -472,11 +259,12 @@ <h2>Scala and Java</h2>
           </div>
         <div class="box single-column">
-          <h1>Commercial Addons</h1>
+          <h1>Akka Enhancements</h1>
```
good

raboof commented:
> For gRPC there is no such page

I guess it doesn't make too much sense for Akka gRPC since that is not just a library, but is always used in combination with a build system plugin
